issue15_code.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct 21 10:10:35 2020
+
+@author: Marit
+"""
+
+from Bio import SeqIO
+from Bio.Seq import Seq
+from Bio import motifs
+import matplotlib.pyplot as plt
+from os import listdir
+from os.path import isfile, join, getsize
+
+
+
+#function
+def motifHistogram(seq1, seq2, directory,nrbins):
+    
+    #-------------------------------------------------------------------------
+    #create motif file of the 2 input motifs forward and revers
+    instances = [Seq(seq1), Seq(seq2)]
+    m = motifs.create(instances)
+    r = m.reverse_complement()
+    #-------------------------------------------------------------------------
+    
+    #get all reads of files in the folder
```

Comment:
The script is doing more than specified, the input that was specified was a set of sequences.